resources/schema.gql

```diff
@@ -1056,6 +1100,23 @@ type TemporalTable {
 	primaryKey: [String!]!
 }
 
+type TimeDelta {
+	every: Int!
+	unit: TimeUnit!
+}
+
+input TimeDeltaInput {
+	every: Int!
+	unit: TimeUnit!
+}
```

Comment:
Ugh, this makes me not want to use GQL for any write operations :(

Reply:
At least we reused `TimeUnit` primitive. But yes, a struct cannot be a `SimpleObject` or `InputObject` at the same time.